docs/spec_draft.md

```diff
@@ -2146,6 +2149,111 @@ where `predicate = rank(pred) == 0 ? pred : pred[i0, ..., iR-1]`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.shift_left
+
+### Semantics
+
+Performs element-wise left-shift operation on the `lhs` tensor by `rhs` number
```

Comment:
What happens in case sign bit overflows? e.g. 1<-0 or 0<-1 for the sign bit? It could be obvious for some, but could also be worth pointing it out that bits are shifted regardless of the signedness. Here is the proposal for the wording and the following ops for consistency. Wdyt?:
```
Performs element-wise signless bit-shift operation on the `lhs` tensor by `rhs`
number of bits to the left and produces a `result` tensor.
```

Reply:
I have covered the case in examples. Since left-shift operation is a standard common operation in most languages wrote it as is.

Reply:
Agreed about keeping "left shift", "arithmetic right shift" and "logical right shift" without explanations, similarly how we are doing this for well-known floating-point functions like `sin` or `cos`.